com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/PathTracing/Shaders/PathTracingMain.raytrace

```diff
@@ -66,12 +77,46 @@ void RayGen()
     jitteredPixelCoord.x += GetSample(currentPixelCoord, _RaytracingSampleIndex, 40);
     jitteredPixelCoord.y += GetSample(currentPixelCoord, _RaytracingSampleIndex, 41);
 
-    // Compute the ray direction, from those coordinates
-    float3 directionWS = -normalize(mul(jitteredPixelCoord, (float3x3)_PixelCoordToViewDirWS));
+    float3 directionWS;
+    float3 cameraPosWS;
+
+    float apertureRadius = _PathTracedDoFConstants.x;
+    if (apertureRadius == 0.0)
+    {
+        // Compute the ray direction from those coordinates (fast path for zero aperture)
+        directionWS = -normalize(mul(jitteredPixelCoord, (float3x3)_PixelCoordToViewDirWS));
+        cameraPosWS = _WorldSpaceCameraPos;
+    }
+    else
+    {
+        // Compute the ray origin and direction for a lens with non-zero aperture
+
+        // Apply the inverse viewport transform to go from vieport coordinates to NDC
+        jitteredPixelCoord = mul(jitteredPixelCoord, (float3x3)_InvViewportTransform);
+
+        // Sample the lens apperture
```

Comment:
Add details about the dimensions used:
40 for path tracing, 2 for sub-pixel jittering, 64 for SSS -> 106, 107